### PR TITLE
caption modifyied

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @KalpanaReddyC
- [ ] @pulipati2021
- [ ] @rajkumarinfomagnus

### PR DESCRIPTION
**Pull request by KalpanaReddyC**
Updated the cation closes #2

### CODE REVIEW

File: _slides/02-KalpanaReddyC.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Awesome caption
```
Comment (v-rajkumarinfomagnus) [inline _slides/02-KalpanaReddyC.md]: Update this to good caption

Comment (v-rajkumarinfomagnus) [inline _slides/02-KalpanaReddyC.md]:
```suggestion
Awesome caption
```
Good caption

**Review by rajkumarinfomagnus — CHANGES_REQUESTED** [review]
review the comments

File: _slides/02-KalpanaReddyC.md (hunk 1 above)
Comment (v-rajkumarinfomagnus) [inline _slides/02-KalpanaReddyC.md]:
```suggestion
perfect caption
```

**Review by rajkumarinfomagnus — CHANGES_REQUESTED** [review]

**Review by rajkumarinfomagnus — APPROVED** [review]

**Pull request merged**